docs/pages/guides/environment-variables.md

```diff
@@ -54,6 +61,31 @@ function Post() {
 }
 ```
 
+</Tab>
+
+<Tab>
+
+For SDK 45 and below, it is available on the `Constants.manifest` property.
```

Comment:
I almost feel like using collapsible panels for 'Using SDK45 or below?' similar to what we have for classic builds etc. This is too distracting having two use cases written out for two different SDK versions side by side.